angular-chart.js

```diff
@@ -362,5 +365,11 @@
       var options = angular.extend({}, Chart.defaults.global, ChartJs.getOptions(type), scope.options);
       return options.responsive;
     }
+
+    function destroyChart(chart, scope) {
+      if(!chart) return;
+      chart.destroy();
+      scope.$emit('destroy', chart);
```

Comment:
Nit: does it make sense to pass the chart reference in this case if it's destroyed?